### PR TITLE
Bump expensify-common version

### LINKED ISSUE #138: chore: bump expensify-common
Version which we are inside inside parser is outdated - Expensify/App has newer version

### PR DESCRIPTION
**Pull request by robertKozik**
### Details
closes #138

### Related Issues

https://github.com/Expensify/react-native-live-markdown/issues/138

### Manual Tests

Verify that app is building correctly and text is being parsed

### Linked PRs
N/A

### CODE REVIEW

File: parser/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#039f4c584655cb4b5e933c9058b84ead7c8c6aee",
```
Comment (tomekzaw) [inline parser/package.json:23]: It looks like Expensify/App uses a different commit: https://github.com/Expensify/App/blob/06c7dac74ad5bcc8009c614570625afe4006f621/package.json#L103

Comment (robertKozik) [reply parser/package.json:23]: Yes, I'm now creating the PR with bumping version in Expensify/App as well so it will be aligned

**Review by tomekzaw — APPROVED** [review]

File: parser/package.json (hunk 1 above)
Comment (robertKozik) [reply parser/package.json:23]: I'll wait to E/App PR to be merged, before this one

**Comment by robertKozik** [comment]
Fixes #179

**Review by tomekzaw — DISMISSED** [review]

**Review by tomekzaw — APPROVED** [review]

**Pull request merged**